internal/tokenizer/flex/tokenize_c.go

```diff
@@ -0,0 +1,71 @@
+package flex
+
+// #include <stdlib.h>
+// #include "linguist.h"
+// #include "lex.linguist_yy.h"
+// int linguist_yywrap(yyscan_t yyscanner) {
+// 	return 1;
+// }
+import "C"
+import "unsafe"
+
+const maxTokenLen = 32 // bytes
+
```

Comment:
I would also propose to include a `go:generate` directive that updates the C tokenizer, if it makes sense.

Reply:
I think I did not communicate this well enough, but the only code that is somehow "generated" in this PR is the C `lex.linguist_yy.{h,c}`, that is vendored inside Linguist and we are using exact copy. 

In the long run, if this tokeniser turns out to be useful beyond classifier accuracy debugging - we could add C code generation script from the flex grammar, but I think it's ok to do that after #193

Reply:
Sorry, I haven't meant to regenerate it. `go:generate` may also be used to `wget` the latest source, for example. Not critical, just a thought.